### PR TITLE
Adding Jess Ingrassillino's post about pycon 2017 finaid

### PR DESCRIPTION
**Pull request by estherbester**

### CODE REVIEW

**Comment by estherbester** [comment]
@jess_ingrass i changed the markdown/yaml formatting because mynt was complaining. does the author_link and everything else look ok?

File: blog/_posts/2017/2017-01-25-pycon-financial-aid-2017.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+WHAT YOU WILL GET IN RETURN?
+---
+
+* Unless you explicitly choose to be anonymous (though the online donation site or in private discussions), we will promote and profusely thank you via our public channels ([Twitter](https://twitter.com/pyladies), this blog, and at our PyCon booth)
+* Provide a detailed write-up and financial introspection of how many women we've helped because of this campaign
+* Visibility! It shows the PyCon/Python community that your company/you are serious about women in tech - which can directly benefit sponsors with increased hiring pool of female Pythonistas.
+
+WAYS TO DONATE
+---
+All of the following methods are tax-deductible with funds going to the [Python Software Foundation](https://www.python.org/psf-landing/), a 501(c)3 charitable organization.
```
Comment (econchick) [inline blog/_posts/2017/2017-01-25-pycon-financial-aid-2017.md]: Just one nitpick - mynt renders the `501(c)3` bit as a copyright sign in the middle:

2017-01-26 at 11 16 am

Unfortunately escaping the parens don't help. Could we just change it to `501c3`?

**Comment by jessingrass** [comment]
LGTM 👍

**Comment by econchick** [comment]
Looks like heroku is having [issues](https://status.heroku.com/incidents/1037) - will rerun build & deploy once it stabilizes.

**Comment by econchick** [comment]
[It's deployed!](http://blog.pyladies.com/pycon-financial-aid-2017/) - go tweet about it @jessingrass @lorenamesa

**Pull request merged**